packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -397,7 +402,15 @@ const postEnterCodeLPA = (views) => {
 		}
 
 		// check token
-		const tokenResult = await isTokenValid(emailCode, id, user.email, req.session, user.lpaCode);
+		const enrolUsers = true;
```

Comment:
Is this not meant to be the flag value? `enrolUsersFlag`?

Reply:
this is for lpas
the entire process relies on the sql user so this should always be true
perhaps it gets reworked if we split out appellant-dashboard and sql users into separate flags